### ISSUE TITLE
Multiple audio languages can't be changed

### ISSUE THREAD

**Issue by ArsenijN** [libre-tube/LibreTube#6315]
### Steps to reproduce

1. Open LibreTube
2. Play some video like MrBeast or something that has multiple audio languages
3. In settings (last icon if press on the displaying video) they show "_some language_ - unknown type of audio track", and when try to change language of audio it doesn't 

### Expected behavior

Change audio language instead of Spanish (doesn't depends on the first language that's shown in the list, Spanish is third on the "Survive 100 Days In Nuclear..." video) or other

### Actual behavior

Video plays on the same language and not changing

### LibreTube version

0.24.0

### Android version

Android 14

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by akano12** [comment]
Can you share a recording? Audio changes fine when trying to reproduce following your steps

**Comment by ArsenijN** [comment]
Yes, sure (sorry for the late answer) https://github.com/user-attachments/assets/13e9df88-eef3-47aa-bc33-fdb59d3e43fb

**Comment by Selectx** [comment]
This happened to me too. The language is set in Spanish and no matter which one I choose, it will not change.

**Comment by finBitorg** [comment]
Same problem

**Comment by Bnyro** [comment]
see https://github.com/libre-tube/LibreTube/issues/7482
